packages/devextreme/js/__internal/ui/form/form.ts

```diff
@@ -1846,8 +1880,14 @@ class Form extends Widget<FormProperties> {
       instruction: item.aiOptions?.instruction,
     }));
 
+    const clipboardText = text ?? await navigator.clipboard?.readText();
```

Comment:
since it's not necessarily a clipboardText, can we name this variable more accurate?
like `smartPasteText`?